spec/features/navigation_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe 'Site Navigation' do
   describe 'As a Visitor' do
-    it 'I see a nav bar with links to specific indexes' do
```

Comment:
Updated tests to include vcr

Reply:
Thanks for being willing to do this, looks repetitive.

Reply:
ditto, thank you so much this will help a lot!